src/Provider/ImageProviderInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Provider;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ *
+ * @phpstan-import-type FormatOptions from MediaProviderInterface
+ */
+interface ImageProviderInterface
+{
+    /**
+     * @return array<string, int|string|bool|array|null>[]
```

Comment:
```suggestion
     * @return array<array<string, int|string|bool|array|null>>
```

I personally dont like mixing both array<> and [].
Here, it's easy to miss it IMHO.

Reply:
Good point, fixed.